### PR TITLE
pretraining: configure hparms for base and large models

### PR DESCRIPTION
**Pull request by stefan-it**
Hi,

this PR adjusts and adds some useful hyper-parameters to the `configure_pretraining.py` script.

One imporant fix is the learning rate for base/large models: originally 2e-5 was used and is now corrected to 2e-4, because this value is used in the paper.

### CODE REVIEW

File: configure_pretraining.py [hunk 1]
```diff
@@ -115,11 +115,14 @@ def __init__(self, model_name, data_dir, **kwargs):
     # paper for the full hyperparameters
     # else:
     #   self.max_seq_length = 512
-    #   self.learning_rate = 2e-5
+    #   self.learning_rate = 2e-4
     #   if self.model_size == "base":
+    #     self.embedding_size = 768
```
Comment (clarkkev) [inline configure_pretraining.py]: this isn't strictly needed because the embedding size is set to the hidden size when self.embedding_size=None, but perhaps it's better to explicitly set it anyway

**Comment by clarkkev** [comment]
Good catch!
